jplag/src/main/java/de/jplag/clustering/Cluster.java

```diff
@@ -15,17 +14,23 @@
     private final float communityStrength;
     private final Collection<T> members;
     private ClusteringResult<T> clusteringResult = null;
+    private final float averageSimilarity;
 
-    public Cluster(Collection<T> members, float communityStrength) {
+    public Cluster(Collection<T> members, float communityStrength, float averageSimilarity) {
```

Comment:
Just for understanding: The average similarity of a cluster is the mean value of all submissions tuple similarities among the submissions in the cluster? Maybe add a JDoc comment for the constructor and clarify it in there. The author of the class did not specify one it seems.

Reply:
Yes, exactly. I added the JDoc.